lib/database/achievement.php

```diff
@@ -498,15 +496,15 @@ function UploadNewAchievement(
             VALUES (
                 NULL, '$gameID', '$title', '$desc',
                 '$mem', '$progress', '$progressMax',
-                '$progressFmt', '$points', '$type',
-                '$author', NOW(), NOW(),
+                '$progressFmt', $points, $type,
+                '$dbAuthor', NOW(), NOW(),
                 NOW(), 0, 0,
                 '$badge', 0, NULL,
                 0
             )";
         // log_sql($query);
-        if (s_mysql_query($query) !== false) {
-            global $db;
```

Comment:
`sanitize_sql_inputs` escapes quotes with backslashes, and `s_mysql_query` rejects anything containing backslashes. calling `mysqli_query` directly after calling `sanitize_sql_inputs` seems common in places where we support apostrophes (messages, comments, forums, etc).

Reply:
as discussed - this is the preferred way 👍